.gitignore

```diff
@@ -1,4 +1,14 @@
 dist/
 node_modules/
-.vscode/
-npm-debug.log
\ No newline at end of file
+npm-debug.log
+
+# Created by https://www.gitignore.io/api/visualstudiocode
+
+### VisualStudioCode ###
+.vscode/*
+!.vscode/settings.json
```

Comment:
This is a good example, but most of the options in that `settings.json` can be covered by an `.editorconfig`.

Having editor-specific settings committed makes me really nervous. If we can make a case for this, we can make a case for committing settings for every editor. That presents a pretty big maintenance headache. If we decide "only settings for VSCode", why is VSCode special?